launcher/ui/connectpage.cpp

```diff
@@ -34,19 +34,34 @@
 
 #include <common/endpoint.h>
 
+#include <QAction>
+#include <QFileInfo>
 #include <QProcess>
 #include <QSettings>
 
+#include <qplatformdefs.h>
+
+#ifdef Q_OS_LINUX
+#include <sys/stat.h>
+#endif
+
 using namespace GammaRay;
 
 ConnectPage::ConnectPage(QWidget *parent)
     : QWidget(parent)
     , ui(new Ui::ConnectPage)
+    , m_localPrefix("local://")
+    , m_tcpPrefix("tcp://")
```

Comment:
Those two don't need to be non-static member variables I think, they seem to be used as constants.